tests/integration-tests/tests/storage/storage_common.py

```diff
@@ -295,21 +295,45 @@ def check_fsx(
     mount_dirs,
     bucket_name,
     headnode_only=False,
+    file_cache_path=None,
 ):
     remote_command_executor = RemoteCommandExecutor(cluster)
     scheduler_commands = scheduler_commands_factory(remote_command_executor)
     fsx_ids = get_fsx_ids(cluster, region)
     logging.info("Checking the length of mount dirs is the same as the length of FSXIDs")
     assert_that(len(mount_dirs)).is_equal_to(len(fsx_ids))
+
+    fsx_lustre_mount_options = "defaults,_netdev,flock,user_xattr,noatime,noauto,x-systemd.automount"
     for mount_dir, fsx_id in zip(mount_dirs, fsx_ids):
         logging.info("Checking %s on %s", fsx_id, mount_dir)
         file_system_type = get_file_system_type(fsx_id, region)
+
         if file_system_type == "LUSTRE":
-            assert_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir, region, fsx_id)
-            if bucket_name:
-                _test_import_path(remote_command_executor, mount_dir)
-                _test_export_path(remote_command_executor, mount_dir, bucket_name, region)
-                _test_data_repository_task(remote_command_executor, mount_dir, bucket_name, fsx_id, region)
+            mount_name = get_mount_name(fsx_id, region)
+            if fsx_id.startswith("fc-"):
+                assert_fsx_lustre_correctly_mounted(
+                    remote_command_executor,
+                    mount_dir,
+                    mount_name=mount_name,
+                    mount_options=fsx_lustre_mount_options + ",x-systemd.requires=network.service",
```

Comment:
*Observation to cover in another PR*: it seems that the mount option `x-systemd.requires=network.service` is required only for FSx FileCache, but it seems reasonable to have it also in pother FSx File Systems.

Reply:
So this change in other PR need to be added in cookbook.